tests/phpunit/unit/Controller/FrontendTest.php

```diff
@@ -405,6 +480,20 @@ public function testSearchListing()
 
         $response = $this->controller()->search($this->getRequest());
 
+        $this->assertTrue($response instanceof TemplateView);
+        $this->assertSame('search.twig', $response->getTemplate());
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testSearchLegacy()
```

Comment:
I've already used the legacy prefix instead of the annotation. Your argument for filter is so you can run `testSearchLegacy` and `testSearchWithFilters` with one filter? Ok :expressionless: